src/components/v5/common/ActionSidebar/partials/forms/SplitPaymentForm/partials/SplitPaymentPayoutsTotal/types.ts

```diff
@@ -0,0 +1,10 @@
+import { type Token } from '~types/graphql.ts';
+
+import { type SplitPaymentRecipientsFieldModel } from '../SplitPaymentRecipientsField/types.ts';
+
+export interface SplitPaymentPayoutsTotalProps {
+  data: SplitPaymentRecipientsFieldModel[];
+  token: Token;
+  moveDecimals?: boolean;
```

Comment:
It's a bit unclear what this does (could be converting either way), could you rename it to make it clear it's converting to WEI or from WEI?